doc/reference/plutus/plutus-minting-script-example.md

```diff
@@ -0,0 +1,83 @@
+# Plutus Minting Scripts
+
+## What is a Plutus minting script?
+
+A Plutus minting script is required to validate the minting of multi-asset tokens. Unlike Mary-era scripts, Plutus scripts support more logic beyond required signatures and timelocks.
+
+### An example of using a Plutus minting script
+
+Below is an example that shows how to use a Plutus minting script. This is a step-by-step
+process involving:
+
++ the creation of the `AlwaysSucceeds` Plutus minting script (i.e. anybody can mint)
++ the creation of a transaction that mints multi-assets using the `AlwaysSucceeds` Plutus minting script
+
+In this example we will use the [anyone can mint](../../../plutus-example/plutus-example/src/Cardano/PlutusExample/MintingScript.hs) Plutus minting script. To execute it, we require the following:
+
+- Collateral tx input(s) that are provided and forfeited if the Plutus script fails to execute.
+- Serialization of the Plutus script in the text envelope format (required for cardano-cli).
+- A redeemer.
+
+#### Creating the `AlwaysSucceeds` Plutus minting script
+
+The plutus-example executable will automagically generate several Plutus scripts in the CLI-compatible text envelope format.
+
+Run the following commands:
+
+```bash
+cd plutus-example
+
+cabal run exe:plutus-example
+```
+
+This will output `anyone-can-mint.plutus` in the `generated-plutus-scripts` dir.
+
+#### Setting up a local Alonzo node cluster
+
+This convenient script will set up an Alonzo cluster immediately on your local machine:
+
+```bash
+cabal install cardano-cli
+cabal install cardano-node
+./scripts/byron-to-alonzo/mkfiles.sh alonzo
+```
+
+Follow the instructions displayed in the terminal to start your Alonzo cluster.
+
+#### Minting multi-assets using the Plutus minting script
+
+To mint a multi-asset, you need the Plutus script policy ID and a redeemer. Because this minting script always succeeds, you can use any redeemer:
+
+```bash
+> cardano-cli transaction policyid --script-file scripts/plutus/scripts/anyone-can-mint.plutus
+> $policyid
```

Comment:
This implies that I'm meant to run a command which is stored in the variable `$policyid` or that both lines are output. It's also shouldn't be labelled as a bash script, since as written it's equivalent to

```bash
transaction policyid --script-file scripts/plutus/scripts/anyone-can-minut.plutus > cardano-cli
```

which is not the intention. Personally I'd prefer just

````
```bash
cardano-cli transaction policyid --script-file scripts/plutus/scripts/anyone-can-mint.plutus
```
````
or if you really want to mention `$policyid`

````
```
$ cardano-cli transaction policyid --script-file scripts/plutus/scripts/anyone-can-mint.plutus
> $policyid
```
````

Reply:
Fair point, I'll make the change.